dist/util/fs.js

```diff
@@ -96,10 +97,7 @@ exports.mkdir = (path) => {
     debug(`mkdir called on ${path}`);
     return new Promise((resolve, reject) => {
         try {
-            return mkdirp_1.default(path, (error) => {
-                if (error) {
-                    return reject(error);
-                }
+            return mkdirp_1.default(path).then(() => {
```

Comment:
We can't accept this change as it is giving below error:
`Expected 2-3 arguments, but got 1.ts(2554)
index.d.ts(11, 38): An argument for 'cb' was not provided.`